infra/gcp/bash/ensure-prod-storage.sh

```diff
@@ -55,6 +55,12 @@ GCR_AUDIT_TEST_PROD_PROJECT=$(k8s_infra_project "prod" "k8s-gcr-audit-test-prod"
 RELEASE_TESTPROD_PROJECT=$(k8s_infra_project "prod" "k8s-release-test-prod")
 RELEASE_STAGING_CLOUDBUILD_ACCOUNT="615281671549@cloudbuild.gserviceaccount.com"
 
+# The staging signer service account. To sign artifacts during staging
+STAGING_SIGNER_SVCACCT="k8s-releng"
+RELEASE_PROCESS_PROJECT="kubernetes-release-test"
```

Comment:
Do we have a reason to use a Google project instead of a community-owned project ? we recently switch for to `k8s-staging-releng-test` https://github.com/kubernetes/release/pull/2414 ?

Reply:
Dropped the google owned project.